dev/org.eclipse.codewind.core/src/org/eclipse/codewind/core/internal/CoreUtil.java

```diff
@@ -152,11 +151,10 @@ public static IPath appendPathWithoutDupe(IPath start, String finish) {
 	 * In: [ "Here", "Is", "Some Input" ]
 	 * Out: "Here, Is, Some Input"
 	 */
-	public static String toCommaSeparatedString(Collection<String> collection) {
+	public static String formatString(String[] strArray, String separator) {
```

Comment:
A simpler implementation for `formatString` is
```
	public static String formatString(String[] strArray, String separator) {
		return Arrays.stream(strArray).collect(Collectors.joining(separator));
	}
```

Reply:
Nice.  Done.